tests/Microsoft.Identity.Test.Unit/CoreTests/RegionDiscoveryProviderTests.cs

```diff
@@ -121,8 +121,15 @@ public async Task ErrorResponseFromLocalImdsAsync()
             }
         }
 
-        private void AddMockedResponse(HttpResponseMessage responseMessage)
+        private void AddMockedResponse(HttpResponseMessage responseMessage, string apiVersion = "2020-06-01", bool expectedParams = true)
```

Comment:
Testing does not seem to cover the fallback?

Reply:
It was in progress. The test to verify this change is added.